modules/compiler/spirv-ll/source/builder.cpp

```diff
@@ -61,78 +62,68 @@ llvm::Value *spirv_ll::Builder::popFunctionArg() {
   return arg;
 }
 
-llvm::DIType *spirv_ll::Builder::getDIType(llvm::Type *type) {
-  const llvm::DataLayout &datalayout =
+llvm::DIType *spirv_ll::Builder::getDIType(spv::Id tyID) {
+  const llvm::DataLayout &DL =
       IRBuilder.GetInsertBlock()->getModule()->getDataLayout();
 
-  uint32_t align = datalayout.getABITypeAlign(type).value();
+  auto *const llvmTy = module.getLLVMType(tyID);
+  auto *const opTy = module.get<OpType>(tyID);
+  assert(llvmTy && opTy && "Could not get type from spv::Id");
```

Comment:
Can you split this into two separate asserts, so that if we get an assertion failure, we know which of the two failed?

Reply:
Yes, good idea. Done.